app/[[...path]]/page.tsx

```diff
@@ -199,10 +199,10 @@ export async function generateMetadata(props: MetadataProps): Promise<Metadata>
     ? `https://${process.env.VERCEL_URL}`
     : domain;
   let title =
-    'Sentry Docs | Application Performance Monitoring &amp; Error Tracking Software';
+    'Sentry Docs | Application Performance Monitoring & Error Tracking Software';
   let customCanonicalTag: string = '';
   let description =
-    'Self-hosted and cloud-based application performance monitoring &amp; error tracking that helps software teams see clearer, solve quicker, &amp; learn continuously.';
+    'Self-hosted and cloud-based application performance monitoring & error tracking that helps software teams see clearer, solve quicker, &amp; learn continuously.';
```

Comment:
```suggestion
    'Self-hosted and cloud-based application performance monitoring & error tracking that helps software teams see clearer, solve quicker, and learn continuously.';
```
Found one more - defer to you on whether this should be "&" or "and" though